content/projects/basic-unit-test-mocks-kata/_index.md

```diff
@@ -111,21 +111,34 @@ If you want some further reading check this out: {{% contentlink path="topics/li
 
 ## The actual project
 
-Write a program that sends a random inspirational quote to an email address, you should export 2 functions from a file named `sendEmail.js`:
-  1. `handleSendEmail` it should take in a parameter which is the recipients email address, and should be the only function responsible for sending the email via SMTP
-  2. `sendEmail` it should handle getting the recipients email address, the email address could come from the command line or passed directly to onSendEmail as a parameter
+Write a program that sends a random inspirational quote to an email address, you should export 2 functions from a file named `send_email.js`:
+
+1. `handleSendEmail`: it should take in a parameter which is the recipients email address, and should be the only function responsible for sending the email via SMTP.
+2. `sendEmail`: it should handle getting the recipients email address, the email address could come from the command line or passed directly to on `sendEmail` as a parameter. It should call `handleSendEmail`.
```

Comment:
I think these instructions probably confuse people. 

It might be good to add a bit of an explanation about why there are 2 functions. Or add some pseudocode or something like that to make it more clear.

Maybe adapt the following:


sendMail is the function that gets called when you execute the script from the command line. It needs to do something like this:

```
emailAddress = get the address from the command line parameters
handleSendMail(emailAddress)
```

The reason we use separate functions is that we want to make sure our functionality is as reusable and testable as possible. If you have one function that grabs the email address from the command line arguments and then immediately sends the email then that function would only be able to be used from the command line. You wouldn't be able to use it in other places.